src/components/onCall/oncall-list-view.tsx

```diff
@@ -22,7 +22,7 @@ interface Props {
  */
 const OnCallListView = ({ selectedDate, setSelectedDate, updatePaidStatus }: Props) => {
   const onCallData = useAtomValue(onCallAtom);
-  const isAccountant = UserRoleUtils.isAccountant();
+  const {isAccountant} = useUserRole();
```

Comment:
Lack of spacing here for curly braces make me wonder if you have the biome linter set up to auto format on save, if not I'd recommend doing so